include/freerdp/settings.h

```diff
@@ -1449,6 +1450,8 @@ struct rdp_settings
 	ALIGN64 BYTE*
 	SettingsModified; /* byte array marking fields that have been modified from their default value */
 	ALIGN64 char* ActionScript;
+
+	ALIGN64 char* GatewayAccessToken; /* 1997 */
```

Comment:
This should be located where the other Gateway settings are. Remove it here, add it there and decrease the padding - otherwise the struct size changes.

Reply:
Ok, done.